src/spec-node/dockerCompose.ts

```diff
@@ -140,7 +151,7 @@ export async function buildAndExtendDockerCompose(config: DevContainerFromDocker
 	// determine base imageName for generated features build stage(s)
 	let baseName = 'dev_container_auto_added_stage_label';
 	let dockerfile = null;
-	const serviceInfo = getInfoForService(composeService);
+	const serviceInfo = getBuildInfoForService(composeService);
 	if (serviceInfo.build) {
 		const { context, dockerfilePath, target } = serviceInfo.build;
 		const resolvedDockerfilePath = cliHost.path.isAbsolute(dockerfilePath) ? dockerfilePath : path.resolve(context, dockerfilePath);
```

Comment:
Can context be relative?

Reply:
Just checked the docs, and it can be. Will update to handle that